lib/content-services/breadcrumb/dropdown-breadcrumb.component.scss

```diff
@@ -59,7 +58,6 @@
 
         &-current-folder {
             text-align: left;
```

Comment:
better remove the 'text-align: left' also --> it causes 'ugly' display on dropdown breadcrumb for the DL from /files